### ISSUE TITLE
feature: add window picker

### ISSUE THREAD

**Issue by ME-Haan** [LazyVim/LazyVim#1083]
### Did you check the docs?

- [X] I have read all the LazyVim docs

### Is your feature request related to a problem? Please describe.

When 'w' is pressed in the Neotree to open a file with window picker (as described in the help menu) I get the following message:
"You'll need to install window-picker to use this command: https://github.com/s1n7ax/nvim-window-picker".

### Describe the solution you'd like

Please add the the necessary plugin (nvim-window-picker or equivalent) for this option to work. 

### Describe alternatives you've considered

None

**Comment by dpetka2001** [comment]
You have the option to add it to your own config.

**Comment by ME-Haan** [comment]
Yes, but I guess it would be logical to include it in LazyVim, since Neotree for example already relies on it. Or maybe add it to the extras for convenience?
In the end I could add every plugin to my config myself, but I use a package like LazyVim for this convenience.

**Comment by dpetka2001** [comment]
There are so many plugins out there that cannot be all installed in a Neovim distro. And many plugins decide to depend on other plugins for some capabilities. A distro generally comes mostly as a template and then users can tinker their personal config however they want. I don't believe it's possible for a distro to cater to everyone's needs. `nvim-cmp` relies on so many external sources for different capabilities. Should they also be included in a distro? I don't think that's possible.

**Comment by folke** [comment]
The solution is to just add the below to any file under `lua/plugins/`:

```lua
return {
    's1n7ax/nvim-window-picker',
    name = 'window-picker',
    event = 'VeryLazy',
    version = '2.*',
    config = function()
        require'window-picker'.setup()
    end,
}
```
